theme/partials/footer.html

```diff
@@ -54,7 +54,7 @@ <h4 class="footer__headline">Support</h4>
                         <h4 class="footer__headline">License</h4>
                         <p>
                             Code licensed under <a
-                                href="{{ config.repo_url }}blob/master/LICENSE.md">BSD 3-Clause</a>.
+                                href="{{ config.repo_url }}/blob/master/LICENSE.md">BSD 3-Clause</a>.
```

Comment:
This link is wrong because we do not longer use a branch with name `master`.

Reply:
Related to #61

Reply:
We need the default branch of the related repository. Example:

https://github.com/laminas/laminas-servicemanager/blob/3.13.x/LICENSE.md

Reply:
@froschdesign I see, would the default branch suffice for licensing? If that's the case, it seems that replacing master with HEAD would do the trick?

As mentioned in [SO post](https://stackoverflow.com/a/65158264/13304566), the url could become: https://github.com/laminas/laminas-servicemanager/blob/HEAD/LICENSE.md

Reply:
Although for these `/blob/` urls a `-` could be used for it to show the default branch, like:
https://github.com/laminas/laminas-servicemanager/blob/-/LICENSE.md
As mentioned in the comments of previous SO post.

Reply:
This means we can also fix the problem with edit link.

https://github.com/laminas/documentation-theme/blob/e53c2d3bd381197a6457238a257385d5d8c5d022/theme/partials/content.html#L38-L49

Reply:
@froschdesign I believe these are the changes that should make everything route to the right place again.